### ISSUE TITLE
Add support for public_html

### ISSUE THREAD

**Issue by neidiom** [systemli/ansible-role-roundcube#37]
Per docs:

```
It is also recommended to change the document root to <install path>/public_html
after installation if Roundcube runs at root of a dedicated virtual host. This
will automatically keep sensitive files out of reach for http requests.
```

https://github.com/roundcube/roundcubemail/blob/master/INSTALL#L193
https://github.com/roundcube/roundcubemail/issues/5707#issuecomment-290662666

**Comment by neidiom** [comment]
@doobry-systemli I think a good way to do this is to add a role variable like use_public_html which if set to true will not create the symbolic link to the current release - https://github.com/systemli/ansible-role-roundcube/blob/main/tasks/main.yml#L64

**Comment by t2d** [comment]
Implemented in #42. Thanks
